tests/test_widerface_format.py

```diff
@@ -398,8 +403,50 @@ def test_inplace_save_writes_only_updated_data(self):
                 ignored_attrs=IGNORE_ALL,
             )
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_convert_to_widerface(self):
```

Comment:
It's not really "conversion", so maybe it should be renamed. Also, it's not really clear why this test is needed, maybe some comments or the test category description should be added.